tgui/packages/tgui/interfaces/RequestsConsole.tsx

```diff
@@ -0,0 +1,46 @@
+import { useBackend } from '../backend';
+import { Button, Stack } from '../components';
+import { Window } from '../layouts';
+import { RequestsData } from './RequestsConsole/Types';
+import { RequestsConsoleHeader } from './RequestsConsole/RequestsConsoleHeader';
+import { RequestMainScreen } from './RequestsConsole/RequestsConsoleMainScreen';
+
+export const RequestsConsole = (props, context) => {
+  const { act, data } = useBackend<RequestsData>(context);
+  const { department } = data;
+  return (
+    <Window title={department + ' Requests Console'} width={500} height={600}>
+      <Window.Content>
+        <RequestsConsoleContent />
+      </Window.Content>
+    </Window>
+  );
+};
+
+export const RequestsConsoleContent = (props, context) => {
```

Comment:
who else is using this? no need to export components used in the same file.
(please don't make me tag all of these, but) there are quite a few other components that should not use `export` in this PR